Dockerfile

```diff
@@ -7,21 +7,16 @@ RUN apt-get update && apt-get install -y \
     xvfb
 
 # Nose is the Python test-runner
-RUN pip install nose nosepipe
+RUN pip install nose nosepipe xvfbwrapper
```

Comment:
About this, have a look at [xvfb-run](http://manpages.ubuntu.com/manpages/xenial/man1/xvfb-run.1.html). It does the same thing, but without infecting our tests. This might limit tests from running on anything but Linux, and adds that additional indentation to a lot of lines.

```bash
$ xvfb-run docker run ...
```